applications/altran-v2x/readme.md

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2020 Altran Technology
+
+Overview of Altran V2X Apps:
+=============================
```

Comment:
This is fine, but it is advised to use markdown syntax for headings, such as: #, ##, ###, ...